doc/GUIDE.md

```diff
@@ -1939,213 +1939,8 @@ can use the `stack dot` command and Graphviz. More information is
 
 ### Travis with caching
 
-Many people use Travis CI to test out a project for every Git push. We have [a
```

Comment:
There are still references to this section, e.g. in https://github.com/commercialhaskell/stack/pull/2578/files#diff-e96fbd8ff8894442360d3de9bdd33895R86, and at least that is now stale even with the redirection.